### DISCUSSION TITLE
Allow the use of related collection fields in layout options of Map, Calendar & Cards

### DISCUSSION THREAD

**Discussion by D3VL-Jack** [directus/directus#19647] [Feature Requests]
### Summary

There's no way to set a related collection field as the for layouts that use a field as an option.

### Basic Example

Take for example a collection `Events`, an `Events` item may have multiple `Times` that the event occurs on.
The user may want to access a calendar of `Events` and be shown all the `Times` an event is occurring.

Or in the case of the Map layout, an `Inspection` may have multiple `Locations`.
The user may want to see a map of `Locations` that are part of the `Inspections`.

In both these cases, the linked collection `Times` or `Locations` are associated to their respective parents with a One to Many, it is also assumed that the `Times` and `Locations` are hidden, to be used only in their linked form.

### Motivation

Within the layouts, this is the only place we cannot use a linked/child collection field - they're available for Template strings and Tabular columns, but not Calendar Start/End, Map Location, Card Image.

"Just use the Map layout on the `Locations` collection, or the Calendar layout on `Times` collection" -  By sending the user to the `Times` collection, rather than the `Events`, when the user clicks a `Times` item on the calendar the user would be taken to that `Times` item, where they'd expect to be taken to the `Event` item.

### before
![image](https://github.com/directus/directus/assets/21148384/80a9f4bf-ed14-4a88-b6ae-9d8600760547)

### after
![image](https://github.com/directus/directus/assets/21148384/776ce834-7799-4841-a7ee-a25eb43f85e6)

### Must Have:
- Ability to select Fields from linked collections within the Layout Options

### Drawbacks

N/A

### Alternatives

N/A

### Adoption Strategy

Non breaking change, this appears to only affect the layout items themselves.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
